src/Illuminate/Database/Events/QueryExecuted.php

```diff
@@ -39,21 +39,30 @@ class QueryExecuted
      */
     public $connectionName;
 
+    /**
+     * The affected row count.
+     *
+     * @var int
+     */
+    public $count;
```

Comment:
Maybe it would be better to name it `rowsCount`? Or even `affectedRowsCount`?

Then we do not need to write a comment, it will be much clear what this variable contains.